components/vf-navigation/vf-navigation.js

```diff
@@ -0,0 +1,47 @@
+// vf-navigation
+// only required for vf-navigation--on-this-page
+
+/**
+ * Function for JS scroll to top functionality
+ * That must be executed exactly once
+ * @example vfNavigationOnThisPage()
+ */
+export function vfNavigationOnThisPage() {
+  var sectionList = document.querySelectorAll("[data-vf-js-navigation-on-this-page-container='true'],[data-vf-js-navigation-on-this-page-container='1']")[0];
```

Comment:
Nonblocking: I wonder if we can just use `querySelector` instead of `querySelectorAll()[0]`

Reply:
It's just the shorthand approach I've been using when there should only be one of a component on a page. Perhaps could be done more elegantly.